hw/top_earlgrey/data/ip/chip_sram_ctrl_testplan.hjson

```diff
@@ -131,7 +131,7 @@
       features: ["SRAM_CTRL.MEMSET"]
       stage: V3
       si_stage: SV3
-      tests: []
+      tests: ["sram_ctrl_memset_test"]
```

Comment:
I think you need to use `bazel: ["//sw/device/tests:sram_ctrl_memset_test"]`, per [this document](https://docs.google.com/document/d/14b1RWc2jozg9CfuFK4xCUMR5j4gCjrgYrif40Irqc7Q/edit)

Reply:
Thanks, updated. It's unclear to me why we have both `tests` and `bazel`, but presumably it's because the tests are dvsim targets and don't have bazel targets?

Reply:
Yes, the dv tests are described in hjson files and not in bazel.